lib/compat/wordpress-5.8/index.php

```diff
@@ -103,3 +103,42 @@ function build_query_vars_from_query_block( $block, $page ) {
 		return $query;
 	}
 }
+
+if ( ! function_exists( 'register_legacy_query_loop_block' ) ) {
+	/**
+	 * Complements the renaming of `Query Loop` to `Post Template`.
+	 * This ensures backwards compatibility for any users running the Gutenberg
+	 * plugin who have used Query Loop prior to its renaming.
+	 *
+	 * @see    https://github.com/WordPress/gutenberg/pull/32514
+	 * @since  5.8.0
+	 * @access private
+	 *
+	 */
+	function register_legacy_query_loop_block() {
```

Comment:
I guess this could be named `gutenberg_*` but it's not that important.

Reply:
Added!